frontend/src/theme/variables.scss

```diff
@@ -91,7 +91,7 @@
     /** secondary **/
     @include color-variables(secondary, #cc0033, #000000);
 
-    --ion-color-tertiary: #6a64ff;
+    --ion-color-tertiary: #5875c7;
```

Comment:
2 issues:
- Please use `@include color-variables(tertiary, #your-color, #contrast-color);` (and replace "your-color" and "contrast-color" :) )
- You just edited the colors for dark mode (in line 76 you see that the block for dark mode starts). Please also add the color for the non-dark-mode.

Reply:
@MBuchalik  I have modified the CSS. Can you check now?

Reply:
Looks good 👍